sdk/search/Azure.Search.Documents/perf/Microsoft.Azure.Search.Perf/Microsoft.Azure.Search.Perf.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why do anything with track 1 assemblies? (Same for file below)

Reply:
Thanks for catching this. This is the project file for the new perf tests for T1 APIs. They snuck in here while I have not finished writing the tests.
I am removing the T1 files from this PR.